.github/workflows/merge_to_master.yml

```diff
@@ -33,6 +33,8 @@ jobs:
           # link vs compile time ssl implementations can break the environment when installing requirements
           # Uninstall pycurl - its likely not installed, but in case the ubuntu-latest packages change
           # Then compile and install it with PYCURL_SSL_LIBRARY set to openssl
+          ## Temprory fix for error in suds-jurko setup command: use_2to3 is invalid.
```

Comment:
This isn't a fix, the version of setuptools should be restricted in requirements, which is still only a workaround.  Install of requirements would still fail trying to build robottelo-container.

Reply:
@mshriver Yeah! I removed it from the code already :)

Reply:
Yeah! The only reason for this w/o cause I believe updating the requirements / fixing the issue in wrapanapi would take time, but since this is fixed in wrapanapi. I have removed this w/o from the code.